dataactbroker/README.md

```diff
@@ -545,15 +545,91 @@ Example output:
   "job_3012_error_url": "https...",
   "job_3006_error_url": "https...",
   "job_3010_error_url": "https...",
-  "job_3008_error_url": "https..."
+  "job_3008_error_url": "https...",
+  "cross_appropriations-program_activity": "https..."
```

Comment:
Thank you for keeping the API docs up to date!

It looks like the _Process Overview_ and _Available Validations_ sections of this doc could also use some updates. Some of those have nothing to do with this PR (e.g., _Available Validations_), but some of the material in _Process Overview_ might be clearer if we mention errors vs warning.

In particular, it might be good to mention which types of schema-based failures result in fatal errors, and which will generate warnings. 

Anyway, if this type of update is out of scope for the story, we can put in a story to address it. What do you think?